Scripts/script-CommonServerPython.yml

```diff
@@ -82,32 +117,61 @@ script: |-
 
 
   def formatEpochDate(t):
+      """
+         Convert a time expressed in seconds since the epoch to a string representing local time
+
+         :type t: ``int``
+         :param t: Time represented in seconds (required)
+
+         :return: A string representing local time
+         :rtype: ``str``
+      """
       if t:
           return time.ctime(t)
       return ''
 
 
   def compareDates(date1, date2, dateFormat):
-    # should return delta between two dates
-    # if instead of date passed string 'now' then will be compared to current time
```

Comment:
-    # if instead of date passed the string 'now' is used then it will be compared to the current time